app/main_dev/launch.js

```diff
@@ -921,12 +921,14 @@ export const launchDex = (walletPath, testnet) => {
     logLevel: 1, // LogLevelDebug
     logPath: logPath,
     logFilename: logFilename
+    // lang: e.g. en-US, pt-BR, etc. (TODO)
```

Comment:
Not sure how to pull in the `locale` from index.js to here.

Reply:
Hey @chappjc
This diff should do the trick:
```
diff --git a/app/actions/DexActions.js b/app/actions/DexActions.js
index 4018f5bd..f7bc87ce 100644
--- a/app/actions/DexActions.js
+++ b/app/actions/DexActions.js
@@ -52,13 +52,14 @@ export const DEX_STARTUP_SUCCESS = "DEX_STARTUP_SUCCESS";
 export const startDex = () => async (dispatch, getState) => {
   dispatch({ type: DEX_STARTUP_ATTEMPT });
   const isTestnet = sel.isTestNet(getState());
+  const locale = sel.currentLocaleName(getState());
   const {
     daemon: { walletName }
   } = getState();
   const walletPath = wallet.getWalletPath(isTestnet, walletName);

   try {
-    const res = await dex.start(walletPath, isTestnet);
+    const res = await dex.start(walletPath, isTestnet, locale);
     dispatch({ type: DEX_STARTUP_SUCCESS, serverAddress: res });
     dispatch(dexCheckInit());
   } catch (error) {
diff --git a/app/main_dev/ipc.js b/app/main_dev/ipc.js
index f79e8076..bf201b74 100644
--- a/app/main_dev/ipc.js
+++ b/app/main_dev/ipc.js
@@ -242,7 +242,7 @@ export const startDcrlnd = async (
   }
 };

-export const startDex = async (walletPath, testnet) => {
+export const startDex = async (walletPath, testnet, locale) => {
   if (GetDexPID()) {
     logger.log(
       "info",
@@ -253,7 +253,7 @@ export const startDex = async (walletPath, testnet) => {
   }

   try {
-    const started = await launchDex(walletPath, testnet);
+    const started = await launchDex(walletPath, testnet, locale);
     return started;
   } catch (e) {
     logger.log("error", `error launching dex: ${e}`);
diff --git a/app/main_dev/launch.js b/app/main_dev/launch.js
index 0c73e3e5..7962d90f 100644
--- a/app/main_dev/launch.js
+++ b/app/main_dev/launch.js
@@ -907,7 +907,7 @@ export const launchDCRLnd = (
 const Mainnet = 0;
 const Testnet = 1;

-export const launchDex = (walletPath, testnet) => {
+export const launchDex = (walletPath, testnet, locale) => {
   if (dex) {
     return;
   }
@@ -920,15 +920,15 @@ export const launchDex = (walletPath, testnet) => {
     net: !testnet ? Mainnet : Testnet,
     logLevel: 1, // LogLevelDebug
     logPath: logPath,
-    logFilename: logFilename
-    // lang: e.g. en-US, pt-BR, etc. (TODO)
+    logFilename: logFilename,
+    lang: locale
   });
   const serverAddress = DEX_LOCALPAGE;
   const sitePath = getSitePath(argv.custombinpath);
   callDEX("startServer", {
     sitedir: sitePath,
-    webaddr: serverAddress
-    // lang: e.g. en-US, pt-BR, etc. (TODO)
+    webaddr: serverAddress,
+    lang: locale
   });
   dex = true;
   return serverAddress;
```

Reply:
Perfect, thanks, @bgptr!

Please not the added comment about the localized_html folder that is used instead of the "templates" folder from 0.2.